doc/tutorials/using_partition_mode_on_nuc.rst

```diff
@@ -176,9 +180,9 @@ Update ACRN Hypervisor Image
    The above command output should contain the ``GRUB`` keyword.
 
 #. Check or update the BDF information of the PCI devices for each
-   pre-launched VM; check it in the ``hypervisor/arch/x86/configs/whl-ipc-i5/pci_devices.h``.
+   pre-launched VM; check it in the ``misc/config_tools/data/whl-ipc-i7/logical_partition.xml``.
```

Comment:
Same here, it's good to do a check but if changes are needed, a copy of the file should be made and used (instead of modifying the original file in the repo).